ibm_catalog.json

```diff
@@ -0,0 +1,87 @@
+{
+  "products": [
+    {
+      "name": "deploy-arch-ibm-access-group",
```

Comment:
This needs to be `deploy-arch-ibm-access-management` as thats the name I submitted for approval